doc/conseil.sql

```diff
@@ -86,7 +86,10 @@ CREATE TABLE public.blocks (
     protocol character varying NOT NULL,
     chain_id character varying,
     hash character varying NOT NULL,
-    operations_hash character varying
+    operations_hash character varying,
+    period_kind character varying,
+    current_expected_quorum integer,
+    active_proposal character varying
```

Comment:
Should we not see the new governance tables here?

Reply:
next PR on top of this one, will publish _very_ soon for review